lib/option.rb

```diff
@@ -52,6 +52,13 @@ def self.kubernetes_locations
     ["almalinux-9", "AlmaLinux 9"]
   ].map { |args| BootImage.new(*args) }.freeze
 
+  VmFamily = Struct.new(:name, :require_shared_slice)
+  VmFamilies = [
+    ["standard", false],
+    ["standard-gpu", false],
+    ["burstable", true]
+  ].map { |args| VmFamily.new(*args) }.freeze
```

Comment:
```suggestion
  ].map { |args| VmFamily.new(*args).freeze }.freeze
```

If you are freezing the array, you probably want to freeze the elements of the array as well.  This is one way to do that, but you could also make `VmFamily` freeze automatically in `initialize`, or use `Data.define` instead of `Struct.new`.

Probably should do similar for the `VmSize`, but since that already exists, might make sense to do that as a separate commit.

Reply:
Changed to Data.define. I agree all other structs in options.rb should be updated accordingly. I can send it later as a separate commit.